### DISCUSSION TITLE
Attempts remaining in playlists are no longer shown in the new design

### DISCUSSION THREAD

**Discussion by 2minwia** [ppy/osu#15272] [Q&A] [answered]
For some reason, in the new playlists design (well, new in airquotes, been a while since it came out by now), the attempts remaining (if set) are not shown anywhere.
It can be quite jarring if you go into a playlist, retry a song a few times, and suddenly not be allowed to play anymore, without any prior indication...
(actually, along with actually showing it in the playlists screen, maybe the "you've retried x times in this session" on the pause screen should be replaced with "attempts remaining: x" for playlists with limited attempts..? Oh Also Also maybe the "Ready to play" text in the lounge could be replaced if you've exhausted your attempts..? okay, i'll stop writing things now. :p)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/15274
